securedrop/journalist_templates/_source_row.html

```diff
@@ -4,27 +4,39 @@
   <time class="date" title="{{ source.last_updated|rel_datetime_format }}" datetime="{{ source.last_updated|rel_datetime_format(fmt="%Y-%m-%d %H:%M:%S%Z") }}">{{ source.last_updated|rel_datetime_format(relative=True) }}</time>
   <div class="designation">
     {% if source.star.starred %}
-      <button class="button-star starred" type="submit"
-              formaction="{{ url_for('col.remove_star', filesystem_id=source.filesystem_id) }}"><i class="fa fa-star"></i></button>
+      <button class="button-star starred" type="submit" formaction="{{ url_for('col.remove_star', filesystem_id=source.filesystem_id) }}">
+        <img src="{{ url_for('static', filename='icons/starred.png') }}" class="icon-drop icon-star-source" width="16" height="15" alt="starred-icon">
+      </button>
       <input type="checkbox" name="cols_selected" value="{{ source.filesystem_id }}">
       <a href="/col/{{ source.filesystem_id }}" id="starred-source-link-{{ loop_index }}" class="code-name {% if source.num_unread != 0 %}unread{% else %}read{% endif %}">
         {{ source.journalist_designation }}
       </a>
     {% else %}
-      <button class="button-star un-starred" type="submit"
-        formaction="{{ url_for('col.add_star', filesystem_id=source.filesystem_id) }}"><i class="fa fa-star"></i></button>
+      <button class="button-star un-starred" type="submit" formaction="{{ url_for('col.add_star', filesystem_id=source.filesystem_id) }}">
+        <img src="{{ url_for('static', filename='icons/unstarred.png') }}" class="icon-drop icon-star-source off-hover" width="16" height="15" alt="unstarred-icon">
+        <img src="{{ url_for('static', filename='icons/starred.png') }}" class="icon-drop icon-star-source on-hover" width="16" height="15" alt="starred-icon">
+      </button>
       <input type="checkbox" name="cols_selected" value="{{ source.filesystem_id }}">
       <a href="/col/{{ source.filesystem_id }}" id="un-starred-source-link-{{ loop_index }}" class="code-name {% if source.num_unread != 0 %}unread{% else %}read{% endif %}">
         {{ source.journalist_designation }}
       </a>
     {% endif %}
   </div>
   <div class="submission-count">
-    <span><i class="far fa-file-archive"></i> {{ ngettext('1 doc', '{num} docs', docs).format(num=docs) }}</span>
-    <span><i class="far fa-file-alt"></i> {{ ngettext('1 message', '{num} messages', msgs).format(num=msgs) }}</span>
+    <span>
+      <img src="{{ url_for('static', filename='icons/files.png') }}" class="icon-drop" width="14" height="16" alt="files-icon">
+      {{ ngettext('1 doc', '{num} docs', docs).format(num=docs) }}
+    </span>
+    <span>
+      <img src="{{ url_for('static', filename='icons/messages.png') }}" class="icon-drop" width="14" height="13" alt="messages-icon">
```

Comment:
Just writing this is causing me some self-loathing, but the messages icon is enough shorter than the files icon that it caught my eye. Compare it with a height of 15. To me they're better aligned, but the stretching does increase the weight of the lines in the messages icon. If no one else sees a need to change it I will happily agree. :man_shrugging: 

Height 13:

![source-list-messages-icon-height-13](https://user-images.githubusercontent.com/47430598/96788331-5627d900-13c1-11eb-92fa-75bd0a827942.png)

Height 15:

![source-list-messages-icon-height-15](https://user-images.githubusercontent.com/47430598/96788340-5aec8d00-13c1-11eb-9a9a-178d19d483b7.png)

Reply:
My own bias is always to respect an icon's ratio if possible, but I'm fine either way. Flagging to @ninavizz for visibility.

Reply:
I had initially put the messages icon the same height as the files icon (16px, natural width of 17.9px) but found it appear much larger than the files icon (14 x 16px). I reduced it, probably too much.

Since this icon is naturally wider than high, I propose to put it at 16px wide and set its height at 14px (natural height would be 14.3px). This will give it the same area as the files icon except in "landscape" mode instead of "protrait" if I can use these terms. Below is a preview of the result:

![image](https://user-images.githubusercontent.com/22901938/96942592-98294b80-14a3-11eb-9d4d-1f60e7bf6cae.png)

Reply:
Above proposal implemented in 3d28682.

Reply:
That looks better to me. @eloquence and @ninavizz?